src/Functions/TransformDateTime64.h

```diff
@@ -89,20 +89,20 @@ class TransformDateTime64
 
 
     template <typename ... Args>
-    inline auto NO_SANITIZE_UNDEFINED execute_extended_result(const DateTime64 & t, Args && ... args) const
+    inline auto NO_SANITIZE_UNDEFINED executeExtendedResult(const DateTime64 & t, Args && ... args) const
```

Comment:
It would be nice if you could apply the same change `execute_extended_result` --> `executeExtendedResult` also in Functions/CustomWeekTransforms.h, Functions/DateTimeTransforms.h and Functions/TransformDateTime64.h. (Feel free to open a new PR for such cosmetic cleanups. Putting that into this PR is also fine.)

Reply:
Just to be sure I verified it today: 
I see that yesterday I already renamed execute_extended_result --> executeExtendedResult in 
CustomWeekTransforms.h, DateTimeTransforms.h and TransformDateTime64.h.

Reply:
Good, thanks.